Tests/StoreKitUnitTests/PurchasesOrchestratorSK1Tests.swift

```diff
@@ -175,10 +280,27 @@ class PurchasesOrchestratorSK1Tests: PurchasesOrchestratorTestCase {
         expect(self.backend.invokedPostReceiptDataCount) == 0
     }
 
+    func testPurchaseSK1ReturnsMissingReceiptErrorIfSendReceiptFailed() async throws {
+        let product = try await self.fetchSk1Product()
+        let payment = self.storeKit1Wrapper.payment(with: product)
+        self.receiptFetcher.shouldReturnReceipt = false
+
+        let (_, _, error, _) = try await withCheckedThrowingContinuation { continuation in
+            self.orchestrator.purchase(
+                sk1Product: product,
+                payment: payment,
+                package: nil,
+                wrapper: self.storeKit1Wrapper
+            ) { transaction, customerInfo, error, userCancelled in
+                continuation.resume(returning: (transaction, customerInfo, error, userCancelled))
+            }
+        }
+        expect(error).to(matchError(ErrorCode.missingReceiptFileError))
+    }
+
     // MARK: - Paywalls
 
-    func testPurchaseSK1PackageWithPresentedPaywall() async throws {
-        self.customerInfoManager.stubbedCachedCustomerInfoResult = self.mockCustomerInfo
```

Comment:
Removed this stub from a bunch of tests, as we are getting the customer info from post receipt stub below, so it is not needed